projects/gloo/cli/pkg/cmd/check/root.go

```diff
@@ -320,6 +347,15 @@ func checkVirtualServices(namespaces, knownUpstreams []string) (bool, error) {
 					}
 				}
 			}
+			acRef := virtualService.GetVirtualHost().GetOptions().GetExtauth().GetConfigRef()
+			if acRef != nil {
+				if !cliutils.Contains(knownAuthConfigs, renderRef(acRef)) {
```

Comment:
This can be collapsed with short-circuit evaluation:
(but don't actually commit this suggestion because it'll have an extra `}`)
```suggestion
			if acRef != nil && !cliutils.Contains(knownAuthConfigs, renderRef(acRef)) {
```